articles/quickstart/webapp/symfony/01-login.md

```diff
@@ -193,18 +186,59 @@ Notice that we need to identify the user provided selected in the step before bo
 
 ## Trigger Authentication
 
-Set the following in `app/resources/views/index.html.twig`
+Set the following in `app/resources/views/default/index.html.twig`
 
 ```html
 {% if app.user %}
     Welcome, {{ app.user.username }}!<br/>
     {{ dump(app.user) }}
     <a href="{{ url('secured') }}">Protected route</a>
-    <a href="{{ logout_url("secured_area") }}">
+    <a href="{{ logoutUrl }}">
         <button>Logout</button>
     </a>
 {% else %}
     <h1>Symfony Auth0 Quickstart</h1>
     <a href="/connect/auth0"><button>Login</button></a>
 {% endif %}
 ```
+
+## Logout
+
+To log the user out, you have to clear the data from the session, and redirect the user to the Auth0 logout endpoint. You can find more information about this in the [logout documentation](/logout).
+
+Add this to your `src/AppBundle/Controller/DefaultController.php`.
+
+```php
+<?php
+
+namespace AppBundle\Controller;
+
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Route;
+use Symfony\Bundle\FrameworkBundle\Controller\Controller;
+
+class DefaultController extends Controller
+{
+    /**
+     * @Route("/", name="homepage")
+     */
+    public function indexAction()
+    {
+        $returnTo = sprintf('%s://%s:%s/auth0/logout',
+            $this->container->get('router')->getContext()->getScheme(),
+            $this->container->get('router')->getContext()->getHost(),
+            $this->container->get('router')->getContext()->getHttpPort());
+        $logoutUrl = sprintf(
+            'https://%s/v2/logout?client_id=%s&returnTo=%s',
+            '${account.namespace}',
+            '${account.clientId}',
+            $returnTo);
+        return $this->render('default/index.html.twig', array(
+            'logoutUrl' => $logoutUrl
+        ));
+    }
+}
+```
+
+::: note
+Please take into consideration that the return to URL needs to be in the list of Allowed Logout URLs in the settings section of the application as explained in [our documentation](/logout#redirect-users-after-logout)
```

Comment:
"Please take into consideration that the return to URL needs to be in the list of Allowed Logout URLs" -> "The return to URL must be in the list of Allowed Logout URLs..."